.github/workflows/node.js.yml

```diff
@@ -0,0 +1,34 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Deployment to AWS lambda
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  serverless-deploy:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x]
```

Comment:
Yes, I think it is overkill here since you're deploying twice. So you'd only release for node version 14.x since it's the last to get deployed..

Reply:
Maybe we could stick to 12.x since that's what we want to deploy to?

Reply:
Agreed